src/UserProfile.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { Card, CardContent, CardHeader, Avatar, Typography, Button } from '@mui/material';
+import { makeStyles } from '@mui/styles';
```

Comment:
I don't know if @nikkigraybeal or @Shauna2022 might know more but this wasn't working for me locally and in the Material-UI docs it seems to indicate that `makeStyles` is legacy / maybe doesn't work the same now.

Reply:
it could be due to the import on your end? do you have the most up to date code on the front end?

Reply:
after a second look i see why its not working. i dont see the file that they are trying to import from

Reply:
@Rjjayy do you have `@muil/styles` installed locally? I was not able to get it working myself, it [says](https://github.com/mui/material-ui/issues/32142) that it's not compatible with React v18 which is what we're using. I'm get curious if there was something you did on your end to get this to work / were you able to render and view your component somehow?

Reply:
If it seems like it'll be difficult to get it to work; I think our recommendation would be to just keep it simple for now and use the unstyled components; or maybe just use plain CSS (or you can use [CSS modules](https://mui.com/material-ui/guides/interoperability/#css-modules)). We can revisit getting the styling working with MUI as a stretch goal but we should try to focus on getting some basic components up and working as we get into the next sprint